archive.py

```diff
@@ -34,6 +34,7 @@ def archive(contacts, now):
             print(archive_msg)
             archive_msg = None
 
+        # assignment causes arching of contact
```

Comment:
The rest of this project uses the convention of full-sentence comments---that means capital letters and periods.